packages/lesswrong/lib/htmlToText.ts

```diff
@@ -0,0 +1,16 @@
+import { compile } from "html-to-text";
+
+const defaultConverter = compile({
+  selectors: [
+    {selector: "a", options: {ignoreHref: true}},
+    {selector: "img", format: "skip"},
+    {selector: "h1", options: {uppercase: false}},
+    {selector: "h2", options: {uppercase: false}},
+    {selector: "h3", options: {uppercase: false}},
+    {selector: "h4", options: {uppercase: false}},
+    {selector: "h5", options: {uppercase: false}},
+    {selector: "h6", options: {uppercase: false}},
+  ],
+});
+
+export const htmlToTextDefault = (html = "") => defaultConverter(html);
```

Comment:
This is what I briefly mentioned in the last dev checkin. We currently call `htmlToText` in a ton of different places in the code with varying options, when in practice we probably always want to use the options here. I've extracted this into its own function, and updated it in a few places where it's obviously correct and also EA Forum specific. It's possible that LessWrong might want different defaults.

Reply:
@jimrandomh I don't think this changes the behavior for LW, since all current uses seem to be specific to the EA Forum, but once we switch to Elastic this is what we'll be using.  Would be good if you could double-check this.  (But this isn't blocking, @oetherington, since we aren't currently using it.)